packages/database/src/sync/buildEncounterLinkedLookupFilter.ts

```diff
@@ -1,12 +1,55 @@
-import { buildEncounterLinkedSyncFilterJoins } from './buildEncounterLinkedSyncFilter';
+import {
+  buildEncounterLinkedSyncFilterJoins,
+  type JoinConfig,
+} from './buildEncounterLinkedSyncFilter';
 import { buildSyncLookupSelect } from './buildSyncLookupSelect';
 import type { Model } from '../models/Model';
 
+export type Options = {
+  extraSelects?: Record<string, string>; // extra selects to attach to the lookup select
+  extraJoins?: JoinConfig[]; // extra joins needed to traverse between this model and the encounters table
+};
+
+/**
+ * Helper function to determine if a facility_id should be populated in sync lookup
+ * Only populates facility_id when the encounter is from a sensitive facility
+ * This ensures sensitive encounters are only synced to their originating facility
+ */
+export function addSensitiveFacilityIdIfApplicable() {
```

Comment:
Could be a const rather than a function - all it does is return the same text every time. Funnily enough I think that will make the usage site less readable, but it is the normal thing to do.